packages/augur-ui/src/modules/market/components/common/comments/market-comments.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class MarketComments extends Component {
+  static propTypes = {
+    marketId: PropTypes.string.isRequired,
+    colorScheme: PropTypes.string.isRequired,
+    dataWidth: PropTypes.string.isRequired,
+    networkId: PropTypes.string.isRequired,
+  };
+
+  static defaultProps = {
+    marketId: PropTypes.string.isRequired,
+    colorScheme: PropTypes.string.isRequired,
+    dataWidth: PropTypes.string.isRequired,
+    networkId: PropTypes.string.isRequired,
+  };
+
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render() {
+    const {
+      marketId,
+      colorScheme,
+      dataWidth,
+      networkId,
+    } = this.props;
+
+    let content = null;
+
+    content = (
+      <div id='fb-comments-container'>
+        <div id='fb-comments' className='fb-comments' data-colorscheme={colorScheme} data-href={'www.augur.net/comments/' + networkId + '/' + marketId} data-width={dataWidth} data-numposts='10'></div>
+      </div>
+    );
+
+    if (!window.document.getElementById('fb-root')) {
+      const fbRoot = window.document.createElement('div');
```

Comment:
Is this needed to create div like this? can just using React work

like const content {
<div id='fb-root'>
</div>
}

return (<> {content}</>)


also where is the marketId ? seems this will only work once? 

also need a constant for the src url since it looks like a magic string to me.

Reply:
I was initially doing it that way and then was trying some different things to see what would allow the comments to load, but I forgot to change it back. Will do that.